code/datums/traits/negative.dm

```diff
@@ -642,6 +642,31 @@
 			return
 		SEND_SIGNAL(quirk_holder, COMSIG_ADD_MOOD_EVENT, "wrong_cigs", /datum/mood_event/wrong_brand)
 
+/datum/quirk/congenital_analgesia
+	name = "Congenital Analgesia"
+	desc = "Due to a rare condition, you have never felt pain. Physical pain, at least. That breakup still hurt."
+	value = -1
+	mob_traits = list(TRAIT_ANALGESIA)
+	gain_text = "<span class='danger'>You've never really felt pain.</span>"
+	lose_text = "<span class='notice'>...Oh god, you're sore.</span>"
+	medical_record_text = "Patient is unable to process pain"
+
+/datum/quirk/congenital_analgesia/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+	H.set_screwyhud(SCREWYHUD_HEALTHY)
+
+/datum/quirk/congenital_analgesia/on_process()
+	. = ..()
+	var/mob/living/carbon/human/H = quirk_holder
+	if(H.hal_screwyhud != SCREWYHUD_HEALTHY)
+		H.set_screwyhud(SCREWYHUD_HEALTHY)
+
```

Comment:
Instead of constantly checking if you still have the thing add a check in set_screwyhud to prevent something from overwriting the quirk